event_test.go

```diff
@@ -63,3 +64,63 @@ func TestEvent_EmbedObjectWithNil(t *testing.T) {
 		t.Errorf("Event.EmbedObject() = %q, want %q", got, want)
 	}
 }
+
+func TestEvent_GetFields(t *testing.T) {
```

Comment:
Don't forget to rename this

Reply:
Done in https://github.com/rs/zerolog/pull/682/commits/f6d674dbec4b018409abf57e38092f1f2b05c83e